doc/tutorials/transformer/masked_modelling.ipynb

```diff
@@ -0,0 +1,277 @@
+{
```

Comment:
<p>I would insert an overall headline. Maybe also a "prerequisites" subtitle before the prerequisites. just to structure the notebook a little better visually.</p><p> </p><p>"Prior to using this notebook the following steps need to be completed: "</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Agreed. Also adding a title makes a notebook look nicer. Will do it in all notebooks.

Reply:
<p>Oh, yes, the typo. Thanks</p>